src/app/public/modules/selection-box/selection-box.component.scss

```diff
@@ -20,11 +20,9 @@
 
   &:focus {
     .sky-switch-control {
-      //default
-      outline: thin dotted;
-
-      //webkit
-      outline: -webkit-focus-ring-color auto 5px;
+      outline: none;
+      border: none;
+      box-shadow: inset 0 0 0 2px $sky-theme-modern-color-blue-74;
```

Comment:
Modern color for default theme?

Reply:
Good catch. That should have only been applied to modern theme. The default-themed checkboxes should not be affected here. Fixed!